Library/Formula/magicrescue.rb

```diff
@@ -0,0 +1,14 @@
+class Magicrescue < Formula
+  desc "Magic Rescue extracts files of known types from a block device."
+  homepage "http://www.itu.dk/people/jobr/magicrescue/"
+  url "http://www.itu.dk/people/jobr/magicrescue/release/magicrescue-1.1.9.tar.gz"
+  sha256 "a920b174efd664afe9760a43700588c9c5e6182cb13d7421e07ab613bceeb3c7"
+
+  def install
+    system "./configure", "PREFIX=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
```

Comment:
Please add a test :). See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!